File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppJava/app/src/main/java/pl/udu/uwr/pum/verynobleappjava/data/laureateresponse/nobelprize/NobelPrize.java
package pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize;

import java.util.ArrayList;

import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.laureate.Link;
import pl.udu.uwr.pum.verynobleappjava.data.laureateresponse.nobelprize.affiliation.Affiliation;

public class NobelPrize {
    public String awardYear;
    public Category category;
    public CategoryFullName categoryFullName;
    public String sortOrder;
    public String portion;
    public String dateAwarded;
    public String prizeStatus;
    public Motivation motivation;
    public int prizeAmount;
    public int prizeAmountAdjusted;
    public ArrayList<Affiliation> affiliations;
    public ArrayList<Link> links;
}

File: 2022-2023/sourcecode/lab/lab11/VeryNobleAppKotlin/app/src/main/java/pl/udu/uwr/pum/verynobleappkotlin/data/laureateresponse/nobelprize/NobelPrize.kt
package pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize

import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.laureate.Link
import pl.udu.uwr.pum.verynobleappkotlin.data.laureateresponse.nobelprize.affiliation.Affiliation

data class NobelPrize(
    val affiliations: List<Affiliation>,
    val awardYear: String,
    val category: Category,
    val categoryFullName: CategoryFullName,
    val links: List<Link>,
    val motivation: Motivation,
    val portion: String,
    val prizeAmount: Int,
    val prizeAmountAdjusted: Int,
    val prizeStatus: String,
    val sortOrder: String
)